### ISSUE TITLE
Logs Panel: snapshot does not respect the time range filter

### ISSUE THREAD

**Issue by socheatsok78** [grafana/grafana#91611]
### What happened?

Creating a dashboard snapshot with Logs panel, when applying time range filter the panel is not updating to reflect the range selected

### What did you expect to happen?

Upon updating the time range, Logs panel should update to reflect to selected time range

### Did this work before?

n/a

### How do we reproduce it?

1. Visit https://snapshots.raintank.io/dashboard/snapshot/S8rtgn0E0oy3Zue7E93OoHNkEHPjnCC7

### Environment (with versions)?

Grafana:
OS:
Browser: 

### Grafana platform?

None

**Comment by socheatsok78** [comment]
!ping to keep this issue alive 

It been awhile since I last checked this, I'll find some time to retest this to see if this issue still relevant.

**Comment by matyax** [comment]
I don't think this is logs related. When changing the time range, I don't see any new request being fired. I'm not sure if this is expected behavior or not @grafana/dashboards-squad
